src/app/public/testing/colorpicker-fixture.ts

```diff
@@ -89,7 +89,7 @@ export class SkyColorpickerFixture {
   public async setValueFromPresets(presetIndex: number): Promise<any> {
     await this.clickColorpickerButtonEl();
 
-    const presetColors = document.querySelectorAll('.sky-preset-color');
+    const presetColors = document.querySelectorAll('.sky-colorpicker-preset-color-area button');
```

Comment:
Not a part of the scope of this PR, but the test fixture spec needed to be updated to be more flexible now that we have modern theme enabled.